app/src/main/res/values-nl/strings.xml

```diff
@@ -1165,6 +1165,10 @@ Helaas mogen er vanwege een Google Play Store-beleid geen links naar donaties in
     <string name="quest_memorialType_stele_wooden">"Houten zuil (gedenkplaat)"</string>
     <string name="quest_memorialType_stele_stone">"Stenen zuil (gedenkplaat)"</string>
     <string name="quest_memorialType_sculpture">"Ander beeldhouwwerk"</string>
+    <string name="quest_moped_prohibited_title">Zijn bromfietsen (klasse b) toegelaten op dit fietspad?</string>
+    <string name="quest_moped_prohibited_designated">Ja, een gebodsbord is gekoppeld aan het "fietspad" bord.</string>
+    <string name="quest_moped_prohibited_allowed">Ja, maar er is geen verbodsbord zichtbaar bij het fietspad.</string>
+    <string name="quest_moped_prohibited_forbidden">Nee, er is een verbodsbord gekoppeld aan het "fietspad" bord.</string>
```

Comment:
Any translated strings will be overwritten on each import from POEditor. So, this should rather be done after merge on POEditor.